vars/applyLandscaperManifests.groovy

```diff
@@ -63,12 +69,22 @@ def call(body) {
                       }
 
                       if(utils.isCD()) {
+                          stage('Create Version') {
+                              dockerImage = "${repoOwner.toLowerCase()}/${imageName}"
```

Comment:
Move all ***Create Version*** stage code in a method because it is replicated at many places